broadinstitute/single_cell_portal_core
# Fix security issue in `ansi-html` (SCP-4191)

This fixes a security vulnerability in [`ansi-html` reported by Dependabot](https://github.com/broadinstitute/single_cell_portal_core/security/dependabot/29).

To test:
* Run `yarn why ansi-html`
* Confirm output includes "We couldn't find a match!"

It satisfies SCP-4191.
